src/Providers/DataGrid/Wijmo/Features/Selection.ts

```diff
@@ -590,15 +590,31 @@ namespace Providers.DataGrid.Wijmo.Feature {
             rowsIndex: number[],
             isSelected = true
         ): number[] {
-            if (this._grid.features.rowHeader.hasCheckbox) {
+            // Check if the row indexes provided exist in the current Grid page.
+            if (
+                rowsIndex.find(
+                    (index) => index >= this._grid.provider.rows.length
```

Comment:
can you store the length in a `const` before doing the find? :)

Reply:
done!